roles/node_exporter/templates/node-exporter.service

```diff
@@ -6,7 +6,7 @@ After=network-online.target
 [Service]
 User=root
 Type=simple
-ExecStart={{ node_exporter_path }} --collector.textfile.directory /var/node_exporter/textfiles --web.listen-address={{ node_exporter_bind_addr }}
```

Comment:
Is this safe to drop?

Reply:
In this context the textfile collector isn't being used, so I'm eliminating it.

However, the web listen address is missing port 9100. I've remedied it in a commit.

I have verified it's working on a test box.